flags.go

```diff
@@ -0,0 +1,57 @@
+package fangs
+
+import (
+	"reflect"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+)
+
+type AddsFlags interface {
```

Comment:
nit: interfaces tend to be named after a role or actor, in this case something like `FlagAdder`